src/test/java/org/retest/rebazer/service/PullRequestLastUpdateStoreTest.java

```diff
@@ -71,7 +71,8 @@ public void getLastDate_should_return_the_date_if_pr_is_handled_before() {
 	}
 
 	@Test
-	public void computeIfAbsent_should_add_value_if_key_not_exist() throws Exception {
+	@SuppressWarnings( "static-method" )
```

Comment:
Random supress warnings, but not on other methods?

Reply:
Needed to prevent compiler warnings in restrictive setup